src/particle/particle/particle.py

```diff
@@ -1022,9 +1011,7 @@ def from_name(cls: Type[Self], name: str) -> Self:
             )  # throws an error if < 1 or > 1 particle is found
             return particle
         except ValueError:
-            raise ParticleNotFound(  # noqa: B904  <- use from None when Python 2 is dropped
-                f'Could not find name "{name}"'
-            )
+            raise ParticleNotFound(f'Could not find name "{name}"') from None
```

Comment:
Better error messages (less unneeded printout)

Reply:
Yep. I confess _I have not yet browsed the code looking for comments such as this one related to Python 2. Good that you looked already 👍.